aptos-move/framework/aptos-framework/sources/Table.move

```diff
@@ -1,152 +1,113 @@
-/// This module provides a temporary solution for tables by providing a layer on top of Vector
+/// Type of large-scale storage tables.
+/// source: https://github.com/move-language/move/blob/1b6b7513dcc1a5c866f178ca5c1e74beb2ce181e/language/extensions/move-table-extension/sources/Table.move#L1
+///
+/// This is a exact copy from the Move repo. It implements the Table type which supports individual table items to
+/// be represented by separate global state items. The number of items and a unique handle are tracked on the table
+/// struct itself, while the operations are implemented as native functions. No traversal is provided.
 module AptosFramework::Table {
     use Std::Errors;
-    use Std::Option;
-    use Std::Vector;
 
-    const EKEY_ALREADY_EXISTS: u64 = 0;
-    const EKEY_NOT_FOUND: u64 = 1;
+    // native code raises this with Errors::invalid_arguments()
+    const EALREADY_EXISTS: u64 = 100;
+    // native code raises this with Errors::invalid_arguments()
+    const ENOT_FOUND: u64 = 101;
+    const ENOT_EMPTY: u64 = 102;
 
-    struct Table<Key: store, Value: store> has store {
-      data: vector<TableElement<Key, Value>>,
+    /// Type of tables
+    struct Table<phantom K, phantom V> has store {
+        handle: u128,
```

Comment:
why we use u128 instead of u64?

Reply:
We actually want to move to U256, especially if you ask @alinush